### DISCUSSION TITLE
Posibility to edit a Tab?

### DISCUSSION THREAD

**Discussion by Hjallis** [sienori/Tab-Session-Manager#962] [Q&A]
Hi, 
Is it possible to edit a saved tab i a saved session list without deleting them and adding a new one? 
if you for example would like to modify the address in one of the tabs you have selected for startup, the only way I have found it is possible to do so is by deleting it and saving it again...

Mush appreciated add-in regardless, thanks! 😊
Sincerely
/H

**Comment by rediffusion** [comment]
LOL it hasn't feature to rename saved tabs😒
